src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMRepository.java

```diff
@@ -1,11 +1,24 @@
 package com.atlassian.bitbucket.jenkins.internal.scm;
 
+import hudson.EnvVars;
+
+import javax.annotation.CheckForNull;
 import javax.annotation.Nullable;
```

Comment:
please remove this annotation. @mhenschke-atl and I have discussed this at length and we've decided that we should standardise on one 'null allowed' annotation. Because of the prevalence of `@CheckForNull` in Jenkins we opted for it. I know this is not your code, but given you're here and altering the constructor where the `Nullable`annotation is, can you please remove it?